config/locales/eng/action.yml

```diff
@@ -1888,7 +1891,9 @@ eng:
     total_cost: "Total cost"
     total_pretax_expenses: "Total pre-tax expenses"
     total_pretax_revenues: "Total pre-tax revenues"
+    # total_purchases: "Total purchases"
     total_quantity: "Total quantity" #?
+    # total_sales: "Total sales"
```

Comment:
Can you add english translations and synchronize other translation with `rake clean`?